client/app/scripts/liveblog-edit/directives.js

```diff
@@ -159,7 +164,12 @@ define([
                         post: '=',
                         onEditClick: '=',
                         allowQuickEdit: '=',
-                        allowUnpublish: '='
+                        reorderPost: '=',
+                        keepHighlighted: '=',
+                        allowUnpublish: '=',
+                        startReorder: '&',
+                        reorder: '&',
+                        index: '='
```

Comment:
All these directive attributes seem to be created to communicate with the controller.
for this purpose it's better to use a service because right now if I need to use this directive, I wouldn't understand what I need to give as attribute values.